### ISSUE TITLE
No applications found

### ISSUE THREAD

**Issue by trochitv** [oliverschwendener/ueli#262]
Hi there!

I would really like to use this application and it works wonderfully on my tower at home. Unfortunately though I am not able to launch any applications with ueli on my surface go. None are listed. I am sure this is an easy fix, but I cannot seam to find the cause of the issue :/

Thanks in advance

Jan

**Comment by Tasogarre** [comment]
@trochitv - Check the indexed locations in your "Application Search" settings.  I had the same issue and then double checked my "Folder Path" listings in the "Application Folders" group and there was a folder that was invalid, so it was causing the index to not list any installed Applications (note: UWP still listed, but no installed apps) even though other folders were accurate.

A common way this can happen is if you use OneDrive - it moves your Desktop folder into a OneDrive subfolder.  I recommend verifying the 3 default "Folder Path" locations (it's easy to copy / paste the location from UELI to File Explorer) and then removing any faulty ones, replacing them with updated and accurate links.  For me, the 1st and 3rd default "Folder Path" were invalid.

@oliverschwendener - It may be helpful to add better error handling here and a notification to the user if one of the indexed folders is invalid / no longer reachable.

Hope this is helpful - it was how I resolved my issues so maybe it will solve yours too since we appeared to have the same problem...
